packages/app/lib/focus-trap-react/index.ts

```diff
@@ -0,0 +1,7 @@
+import FocusTrapWeb from "./index.web";
```

Comment:
Not sure about this. This is what we want to avoid actually. If you are loading this from the native app, I'm afraid it won't work.

Can you do `import type { Props } from "focus-trap-react"` instead?